src/modules/authentacation/components/ResetPassword/ResetPassword.tsx

```diff
@@ -1,7 +1,154 @@
-import React from 'react'
+
+import React, { useState } from 'react'
+import { useForm } from 'react-hook-form'
+import { EMAIL_VALIDATION, PASSWORD_VALIDATION } from '../../../../services/validation/validation'
+import { axiosInstance,  USERS_URLS } from '../../../../services/apisUrls/apisUrls'
+import { useLocation, useNavigate } from 'react-router-dom'
+import { toast } from 'react-toastify';
+
+
+import { FaEyeSlash, FaRegEye, FaRegEyeSlash, FaSpinner } from 'react-icons/fa'
+import { IoEyeOutline } from 'react-icons/io5'
+
+
+interface FormInputs {
+  email: string;
+  password: string;
+  confirmPassword: string;
+  seed: string; // OTP
+}
+
 
 export default function ResetPassword() {
+  const navigate = useNavigate()
+  const [isPasswordVisable, setIsPasswordVisable] = useState(false);
+  const [isConfirmPasswordVisable, setIsConfirmPasswordVisable] = useState(false);
+  const location = useLocation()
+  let { register, formState: { errors, isSubmitting }, handleSubmit } = useForm<FormInputs>({ defaultValues: { email: location.state }, mode: 'onChange' })
```

Comment:
I suggest using a code formatter like prettier, it will automatically format the code for a better and consistent look.